inst/CITATION

```diff
@@ -0,0 +1,17 @@
+# Code copied from citation()
+author <- meta$`Authors@R`
+if (length(author)) {
+    aar <- .read_authors_at_R_field(author)
+    author <- Filter(function(e) {
+        !(is.null(e$given) && is.null(e$family)) && !is.na(match("aut",
+            e$role))
+    }, aar)
+}
+
+bibentry(
+  bibtype = "Manual",
+  title   = paste0(meta$Package, ": ", gsub("[[:space:]]+", " ", meta$Title)),
+  author  = author,
+  year    = format(meta$Date, "%Y"),
```

Comment:
Would this be a suitable short-term fix? `format(Sys.Date(), "%Y")`

Reply:
No, because we want to use the date field if present.